data/coalition/coalition news

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2021 by Arachi-Lover
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+
+news "heliarch news"
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Heliarch broadcast"
+	message
+		word
+			"A sizable crowd is watching one of the monitors in the spaceport, where an interviewer is speaking to a Heliarch agent. You learn from someone in the crowd that the agent was until recently a professional athlete."
+			"A news broadcast shows footage of many Heliarch agents arresting a group of terrorists. The Heliarch who was supervising the operation is later interviewed, explaining that the now prisoners were found to have stolen government equipment."
+
+news "heliarch propaganda"
+	location
+		government "Coalition" "Heliarch"
+	name
+		word
+			"Heliarch propaganda"
+	message
+		word
+			"A short film plays on a monitor during a commercial break. A young Saryd walks in a forest, looking to his sides every few seconds to take in the view. Eventually he turns to see another Saryd, both surprised. As he approaches her and the two start talking, the camera pans out to a rising sun over the forest."
+			"A local screen shows a stressed Saryd working at a computer late at night. The door creaks, and she looks back to find her children peeking. She puts them to sleep, and then goes to bed herself. After waking up, she finds nobody around the house. She goes outside, seeing the children, along with their father, admiring the sun rising."
+			"After a government broadcast, a timelapse plays on a spaceport monitor, showcasing Ring of Friendship gaining more and more solar panels on its exterior over the centuries."
+
+news "longcow rancher"
+	location
+		attributes "arach" "longcows"
+	name
+		word
+			"Longcow rancher"
+	message
+		word
+			`"Caring for longcows, hard work it is. Too much space they take up. Too many legs."`
+			`"Never been around an angry longcow, have you? Word of advice, avoid their sides you should."`
+			`"Stopped using vaccines on longcows, long ago we did. Remember which leg to apply them on, we never could."`
+			`"Wish understand my jokes, my beauties could. Many kicks out of them, they would get."`
+			`"Fresh steak here! First leg steak, second leg steak... a discount for tenth leg steak there is!"`
```

Comment:
This implication of different legs having different qualities is pretty interesting. Neat little alien thing, I like it